modules/installation-azure-config-yaml.adoc

```diff
@@ -41,9 +41,13 @@ controlPlane: <2>
   name: master
   platform:
     azure:
+      encryptionAtHost: true
```

Comment:
You can have encryption at host without disk encryption and vice versa.

Reply:
Thanks John, can you help me understand how this parameter fits into the procedure of this feature? When should customers be advised to enable or disable this parameter?

Reply:
This explains this better than I can: https://www.sebhook.com/2021/01/28/azure-host-based-encryption/

Reply:
I have updated the description of this parameter to say:
"Enables host-level encryption for compute machines. You can enable this encryption alongside user-managed server-side encryption. This feature encrypts temporary, ephemeral, cached and un-managed disks on the VM host. This is not a prerequisite for user-managed server-side encryption." Is this acceptable?

Reply:
Sounds good.